crates/noirc_evaluator/src/ssa/acir_gen/internal_var.rs

```diff
@@ -207,19 +184,4 @@ mod tests {
             }
         }
     }
-    #[test]
-    fn internal_var_from_witness() {
-        let mut evaluator = Evaluator::default();
-
-        let expected_witness = Witness(1234);
-        // Initialize an InternalVar with a `Witness`
-        let mut internal_var = InternalVar::from_witness(expected_witness);
-
-        // We should get back the same `Witness`
-        let got_witness = internal_var.get_or_compute_witness(&mut evaluator, false);
```

Comment:
Yep, since we no longer have `get_or_compute_witness` we can remove this test

Reply:
The function is still here, but it is now the responsibility of the internal_var_cache, because it needs to update the cache when an internal var is modified.
Therefore, the test is still here as well and was moved to internal_var_cache.rs